src/clover/fileparser.py

```diff
@@ -37,6 +37,7 @@
     DesalinationScenario,
     DieselMode,
     EXCHANGER,
+    Grid,
```

Comment:
The `fileparser` module will likely need to import the `GridType` and `GridTier` classes as well here 😄

Reply:
Added